wordpress-mobile/gutenberg-mobile
# Fix Media & Text block crash

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/1650

`Gutenberg` PR -> https://github.com/WordPress/gutenberg/pull/18907

To test:
- Open the Gutenberg **Web** editor:
- Create a new post with this code:
```

<div class="wp-block-media-text alignwide is-stacked-on-mobile is-image-fill"><figure class="wp-block-media-text__media" style="background-image:url(https://geriuxx-atomic-site-1.blog/wp-content/uploads/2019/12/img_0004-1.jpg);background-position:52% 12%"><img src="https://geriuxx-atomic-site-1.blog/wp-content/uploads/2019/12/img_0004-1.jpg" alt="" class="wp-image-29"/></figure><div class="wp-block-media-text__content">
<p class="has-large-font-size"></p>
</div></div>

```
- Save
- Open the newly created post from the Main Apps with metro running.
- Expect the `Media & Text` block to be visible without any crashes.

I'm sharing the snippet to test because there's a bug in the web editor that makes the block invalid when the setting `Crop image to fill entire column` is turned on, due to rounding issues. It has been fixed in master already https://github.com/WordPress/gutenberg/pull/18765 but it is not on production yet.
